Examples/HttpAnalyzer/main.cpp

```diff
@@ -94,7 +95,7 @@ void printUsage()
 			"-------------------------\n"
 			"%s [-hvld] [-o output_file] [-r calc_period] -i interface\n"
 			"\nOptions:\n\n"
-			"    -i interface   : Use the specified interface. Can be interface name (e.g eth0) or interface IPv4 address\n"
+			"    -i interface   : Use the specified interface. Can be interface name (e.g eth0) or interface IPv4 address with port (e.g 192.168.1.1 or 192.168.1.1:80)\n"

```

Comment:
adding `:<port_number>` to the IP address can be less intuitive to many users. Why not add an additional command line parameter (`-p` maybe?) that will be optional and will contain the port number?

This parameter can have a default value of `80`

Reply:
how about sp  and dp? (src port number and dest port number)?
May I add a function about "http wrtier filter"  ?   or  create another example is better?

Reply:
I'd strongly recommend to keep with only dst port. 

You can add an additional command-line parameter for filtering the traffic, same as the `-i` parameter in `PcapPrinter`:

https://github.com/seladb/PcapPlusPlus/tree/master/Examples/PcapPrinter